python/lsst/sconsUtils/tests.py

```diff
@@ -127,18 +127,26 @@ def messages(self, test):
                     "false", "failed")
 
     def run(self, fileGlob):
+        """Create a test target for each file matching the supplied glob.
+        """
```

Comment:
I don't object to this style but I think it is inconsistent with the rest of the package?